include/translated-object.php

```diff
@@ -26,7 +29,7 @@ abstract class PLL_Translated_Object extends PLL_Translatable_Object {
 	 *
 	 * @since 1.8
 	 *
-	 * @param PLL_Model $model Instance of `PLL_Model`.
+	 * @param PLL_Model $model Instance of `PLL_Model`, passed by reference.
```

Comment:
I see that you modified the comment. Wouldn't this work if we simply remove the reference instead?

Reply:
Done in [1e8dabc](https://github.com/polylang/polylang/pull/1503/commits/1e8dabc6f3f835db2659d7f5047b063e8a832151).